packages/riipen-ui-docs/src/modules/components/MarkdownPage.jsx

```diff
@@ -11,141 +18,174 @@ import Menu from "src/modules/components/Menu";
-                } catch (err) {
-                  console.error(err);
-                  return null;
-                }
-
-                const name = options.demo;
-
-                if (!demos || !demos[name]) {
-                  throw new Error(
-                    [
-                      `Missing demo: ${name}. You can use one of the following:`,
-                      Object.keys(demos)
-                    ].join("\n")
-                  );
-                }
-
-                return (
-                  <Demo key={index} demo={demos[name]} options={options} />
+          </Popover>
+        </div>
+      </AppBar>
+      <div className="page">
+        <div className="menu">
+          <Menu />
+        </div>
+        <div className="content">
+          {sections.map((section, index) => {
+            if (demos && constants.REGEX.DEMO.test(section)) {
+              let options;
+
+              try {
+                options = JSON.parse(`{${section}}`);
+              } catch (err) {
+                console.error(err);
+                return null;
+              }
+
+              const name = options.demo;
+
+              if (!demos || !demos[name]) {
+                throw new Error(
+                  [
+                    `Missing demo: ${name}. You can use one of the following:`,
+                    Object.keys(demos)
+                  ].join("\n")
                 );
               }
 
-              return <MarkdownElement key={index} text={section} />;
-            })}
-            <div className="footer">
-              <Divider />
-              <Footer />
-            </div>
-          </div>
-          <div className="toc">
-            <TOC sections={sections} />
+              return <Demo key={index} demo={demos[name]} options={options} />;
+            }
+
+            return <MarkdownElement key={index} text={section} />;
+          })}
+          <div className="footer">
+            <Divider />
+            <Footer />
           </div>
         </div>
-        <style jsx>{`
-          .page {
-            display: flex;
-            justify-content: space-between;
-            padding-top: 22px;
-            position: relative;
-            top: 50px;
-            width: 100%;
-          }
-          .page :global(*:target) {
-            padding-top: 50px;
-          }
-          .content {
-            padding: 0 20px;
-            overflow-x: auto;
-            width: 100%;
-          }
-          .menu {
-            padding: ${theme.spacing(2)}px;
-          }
+        <div className="toc">
+          <TOC sections={sections} />
+        </div>
+      </div>
+      <style jsx>{`
+        :global(.appBar) {
+          justify-content: space-between;
+        }
+        .page {
+          display: flex;
+          justify-content: space-between;
+          padding-top: 22px;
+          position: relative;
+          top: 50px;
+          width: 100%;
+        }
+        .page :global(*:target) {
```

Comment:
I think instead of all these `:global` tags, we actually want to use https://github.com/vercel/styled-jsx#the-resolve-tag.

There's a bunch of examples of that being used in the actually `riipen-ui` package for reference.